Source/src/resources/ResourceMaterial.cpp

```diff
@@ -301,3 +311,26 @@ void Hachiko::ResourceMaterial::UpdateMaterial()
 
     App->scene_manager->GetActiveScene()->RebuildBatches();
 }
+
+void Hachiko::ResourceMaterial::ShaderOption(Program::PROGRAMS value)
+{
+    bool is_selected = (shader == value);
+    if (ImGui::Selectable(ShaderName(value), is_selected))
+    {
+        shader = value;
+    }
+    if (is_selected)
+    {
+        ImGui::SetItemDefaultFocus();
+    }
+}
+
+const char* Hachiko::ResourceMaterial::ShaderName(Program::PROGRAMS value) 
+{
+    switch (value)
+    {
+        case Hachiko::Program::PROGRAMS::DEFERRED_GEOMETRY:
+            return "STANDARD";
+    }
+    return nullptr;
+}
```

Comment:
What's the point of this function?

Reply:
That was a part of the implementation of Materials with diferent shaders so we could show a more comprensive name than the enum, i left it there just in case but we could just remove it.